doc/nrf/ug_matter_hw_requirements.rst

```diff
@@ -162,3 +166,299 @@ Values are provided in kilobytes (KB).
+      | Memory region (incl. size)     | Partition                | Offset         | Size              | Partition elements  | Element offset  | Element size    |
+      +================================+==========================+================+===================+=====================+=================+=================+
+      | External flash (QSPI)          | mcuboot_secondary        | 0kB (0x0)      | 976kB (0xf4000)   | -                   | -               | -               |
+      | (0x800000 = 8192kB)            +--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      |                                | external_flash           | 976kB (0xf4000)| 7216kB (0x70c000) | -                   | -               | -               |
+      +--------------------------------+--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      | Internal flash                 | mcuboot                  | 0kB (0x0)      | 28kB (0x7000)     | -                   | -               | -               |
+      | (0x100000 = 1024kB)            +--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      |                                | mcuboot_primary          | 28kB (0x7000)  | 976kB (0xf4000)   | mcuboot_pad         | 28kB (0x7000)   | 512B (0x200)    |
+      |                                |                          |                |                   +---------------------+-----------------+-----------------+
+      |                                |                          |                |                   | mcuboot_primary_app | (0x7200)        | 975kB (0xf3e00) |
+      |                                +--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      |                                | factory_data             | (0xfb000)      | 4kB (0x1000)      | -                   | -               | -               |
+      |                                +--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      |                                | settings_storage         | (0xfc000)      | 16kB (0x4000)     | -                   | -               | -               |
+      +--------------------------------+--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+      | SRAM primary (0x40000 = 256kB) | sram_primary             | (0x20000000)   | 256kB (0x40000)   | -                   | -               | -               |
+      +--------------------------------+--------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+
+      .. code-block:: console
```

Comment:
Just have a question. Those ASCII art memory maps are just for reference or do you plan to have them in doc?

Personally, I do like to have one and consistent representation - and subjectively I would prefer the table only. As far as i understand the ASCII tables below could be generated by using `west` command (or check in build artifacts).. so maybe instead we can document somewhere the command that can be used to display the memory map.

Reply:
I was just checking the formatting of the table before removing all ASCIIs. If you like the table, I'll remove them.